runtime/browser/demo/demo.js

```diff
@@ -88,16 +88,14 @@ class DemoFlow extends DemoBase {
   async didMount() {
     let root = '../../';
     let loader = new BrowserLoader(root);
-    let {context} = await demoContext({
+    let {arc} = await demoContext({
       loader,
       pecFactory: require('../worker-pec-factory.js').bind(null, root),
       slotComposer: new SlotComposer(this.$('[particle-container]'),  /* affordance */ "dom")
```

Comment:
do we even need demoContext at all?
the only thing it really does now is load the manifest.